kubernetes/samples/scripts/create-fmw-infrastructure-domain/domain-home-in-image/common/Dockerfile

```diff
@@ -0,0 +1,96 @@
+
```

Comment:
why are we replicating this here instead of using the one in docker-images?

Reply:
As mentioned, the current docker file dose not support Support following Arguments 

ARG CUSTOM_EXPOSE_T3_CHANNEL="${CUSTOM_EXPOSE_T3_CHANNEL:-false}"
ARG CUSTOM_T3_PUBLIC_ADDRESS="${CUSTOM_T3_PUBLIC_ADDRESS:-InfraAdminContainer}"
ARG CUSTOM_T3_CHANNEL_PORT="${CUSTOM_T3_CHANNEL_PORT:-30012}"
ARG CUSTOM_FMW_DOMAIN_TYPE="${CUSTOM_FMW_DOMAIN_TYPE:-JRF}" 

to enable exposure of t3Channel  and usage of single py script to create both JRF and Restricted JRF domain